ffmpeg/cppbuild.sh

```diff
@@ -43,6 +45,10 @@ XML2=libxml2-2.9.12
 LIBSRT_VERSION=1.5.0
 WEBP_VERSION=1.2.4
 FFMPEG_VERSION=5.1.2
+LIBSRT_VERSION=1.4.4
+WEBP_VERSION=1.2.2
```

Comment:
Do we need to use old versions of libsrt and webp to make this work?

Reply:
I think just bad merge on my part. Will try cleaning that up.

Reply:
Looks like its still OK.